client/blocks/inline-help/popover-content.scss

```diff
@@ -0,0 +1,168 @@
+.inline-help__popover-content {
+	background-color: var( --color-surface );
+
+	p {
+		font-size: $font-body-extra-small;
+
+		@include breakpoint-deprecated( '>660px' ) {
+			font-size: $font-body-small;
+		}
+	}
+
+	a,
+	button {
+		font-size: $font-body-extra-small;
+
+		@include breakpoint-deprecated( '>660px' ) {
+			font-size: $font-body-small;
+		}
+	}
+
+	&.is-secondary-view-active {
+		.inline-help__search {
+			display: none;
+		}
+
+		.inline-help__secondary-view {
+			display: block;
+		}
+
+		.inline-help__footer {
+			.inline-help__more-button,
+			.inline-help__contact-button {
+				display: none;
+			}
+
+			.inline-help__cancel-button {
+				display: block;
+			}
+		}
+	}
+}
+
+.inline-help__secondary-view {
+	display: none;
+
+	box-sizing: border-box;
+	max-height: 70vh;
+	min-height: 206px;
+	overflow-y: auto;
+	padding: 16px;
+
+	.button {
+		width: 100%;
+		text-align: center;
+	}
+}
+
+.inline-help__view-heading {
+	display: block;
+	font-size: $font-body-small;
+	font-weight: 600;
+	margin-bottom: 5px;
+}
+
+.inline-help__richresult {
```

Comment:
This one doesn't seem to be used right now. Actually, it seems like it was never used since it was introduced in #24390.

Reply:
This is used, just like the the `inline-help__contact` class. It's one of the secondary views styled as `inline-help__${ this.state.activeSecondaryView }`